scripts/gha/test_simulator.py

```diff
@@ -76,64 +100,89 @@ def main(argv):
 
   current_dir = pathlib.Path(__file__).parent.absolute()
   testapp_dir = os.path.abspath(os.path.expanduser(FLAGS.testapp_dir))
-  gameloop_project = os.path.join(current_dir, FLAGS.gameloop_project)
-  ios_device = FLAGS.ios_device
-  device_info = ios_device.split("-")
-  device_name = device_info[0]
-  device_os = device_info[1]
-
-  config_path = os.path.join(current_dir, "integration_testing", "build_testapps.json")
-  with open(config_path, "r") as config:
-    config = json.load(config)
-
-  if not config:
-    logging.info("No config found")
-    return 1
-
-  logging.info("Config found: %s", config)
-
-  testapps = []
-  for file_dir, directories, _ in os.walk(testapp_dir):
+  
+  ios_testapps = []
+  android_testapps = []
+  for file_dir, directories, file_names in os.walk(testapp_dir):
     # .app is treated as a directory, not a file in MacOS
     for directory in directories:
       full_path = os.path.join(file_dir, directory)
       if "simulator" in full_path and directory.endswith(".app"):
-        testapps.append((_get_bundle_id(full_path, config), full_path))
+        ios_testapps.append(full_path)
+    for file_name in file_names:
+      full_path = os.path.join(file_dir, file_name)
+      if file_name.endswith(".apk"):
+        android_testapps.append(full_path)    
 
-  if not testapps:
+  if not ios_testapps and not android_testapps:
     logging.info("No testapps found")
     return 1
 
-  logging.info("Testapps found: %s", "\n".join(path for _, path in testapps))
+  tests = []
+  if ios_testapps:
+    logging.info("iOS Testapps found: %s", "\n".join(path for path in ios_testapps))
+    
+    ios_device = FLAGS.ios_device
+    device_info = ios_device.split(";")
+    device_name = device_info[0]
```

Comment:
check if list has 2 elements after splitting to avoid IndexOutOfBoundsException.